docs/getting_started/quickstart.rst

```diff
@@ -0,0 +1,165 @@
+Getting Started
+===================
+
+.. note:: 
+  | *Estimated Reading Time: 10 minutes*
+  | You can follow along this tutorial in a Jupyter notebook `here <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_. 
```

Comment:
Getting another "Page not found" here for me.

Reply:
Need to wait until PR is pulled in for notebook link to work